source/architecture.rst

```diff
@@ -136,8 +136,7 @@ like templates, devices and flows.
 Cron
 ****
 
-Cron is a dojot's microservice that allows you to schedule events (or requests) to
-be emitted (done) to other microservices inside dojot platform.
+Cron is a dojot's microservice that  allows you to schedule events to be emitted - or requests to be sent - to other other microservices inside dojot platform.
```

Comment:
> [...] or requests to be sent - to **other other** microservices [...]

Reply:
And there is an exceeding space in `that  allows`